ext/pdo_odbc/odbc_driver.c

```diff
@@ -350,6 +350,20 @@ static bool odbc_handle_set_attr(pdo_dbh_t *dbh, zend_long attr, zval *val)
 	}
 }
 
+static int pdo_odbc_get_info_string(pdo_dbh_t *dbh, SQLUSMALLINT type, zval *val)
+{
+	RETCODE rc;
+	SQLSMALLINT out_len;
+	char buf[256];
+	pdo_odbc_db_handle *H = (pdo_odbc_db_handle *)dbh->driver_data;
+	rc = SQLGetInfo(H->dbc, type, (SQLPOINTER)buf, 256, &out_len);
+	/* returning -1 is treated as an error, not as unsupported */
+	if (rc == -1)
```

Comment:
I think we should use the macros here, so
```suggestion
	if (rc == SQL_ERROR)
```
Maybe better:
```suggestion
	if (rc != SQL_SUCCESS && rc != SQL_SUCCESS_WITH_INFO)
```
And in any way, please enclose the if-body in braces.